test/functional/fixtures/page-js-errors/testcafe-fixtures/test-controller.js

```diff
@@ -0,0 +1,62 @@
+const { CLIENT_ERROR_MESSAGE, CLIENT_PAGE_URL, CLIENT_PAGE_URL_REGEXP } = require('../constants');
+
+fixture`TestController method`
+    .page('http://localhost:3000/fixtures/page-js-errors/pages/skip-js-errors.html');
+
+test('Should skip JS errors with boolean param', async t => {
+    await t.skipJsErrors(true)
+        .click('button');
+});
+
+test('Should skip JS errors with multiple options', async t => {
+    await t.skipJsErrors({
+        message: CLIENT_ERROR_MESSAGE,
+        pageUrl: CLIENT_PAGE_URL,
+    })
+        .click('button');
+});
+
+test('Should skip JS errors with callback function', async t => {
+    await t.skipJsErrors(({ message, pageUrl }) => message === CLIENT_ERROR_MESSAGE && pageUrl === CLIENT_PAGE_URL,
+        { CLIENT_PAGE_URL, CLIENT_ERROR_MESSAGE },
+    )
+        .click('button');
+});
+
+test('Should fail with callback function', async t => {
+    await t.skipJsErrors(({ message, pageUrl }) => message === CLIENT_ERROR_MESSAGE && pageUrl === 'incorrect url',
+        { CLIENT_ERROR_MESSAGE })
+        .click('button');
+});
+
+test("Should fail if message option doesn't satisfy the client error message", async t => {
+    await t.skipJsErrors({
+        message: /incorrect message/,
+        pageUrl: CLIENT_PAGE_URL_REGEXP,
+    })
+        .click('button');
+});
+
+test('Should fail due to error in callback function', async t => {
+    await t.skipJsErrors(({ message }) => message === CLIENT_ERROR_MESSAGE)
```

Comment:
At first glance, it's not clear that this callback function raises an error.
Replace with:
```js
await t.skipJsErrors(() => throw new Error('Error in the skipJsError callback function'))
```